Sources/StreamChat/Database/DTOs/MemberModelDTO.swift

```diff
@@ -76,6 +76,14 @@ extension MemberDTO {
         new.id = Self.createId(userId: id, channeldId: channelId)
         return new
     }
+    
+    static func loadLastActiveMembers(cid: ChannelId, context: NSManagedObjectContext) -> [MemberDTO] {
+        let request = NSFetchRequest<MemberDTO>(entityName: MemberDTO.entityName)
+        request.predicate = NSPredicate(format: "ANY channel.cid == %@", cid.rawValue)
+        request.sortDescriptors = [ChannelMemberListSortingKey.lastActiveSortDescriptor]
+        request.fetchLimit = context.channelConfig?.lastActiveMembersLimit ?? 25
```

Comment:
IMHO context shouldn't be aware of `channelConfig` (I even don't think there should be a `channelConfig`, but that was another comment), `loadLastAciveMembers` should accept a `limit` param and it should be passed down (possibly from `ChannelController`, `itemCreator` block?)